src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs

```diff
@@ -553,8 +553,15 @@ private void AttributePrefix(IEnumerable<HtmlSymbol> whitespace, IEnumerable<Htm
             }
             else
             {
+                // Output the attribute name, the equals and optional quote. Ex: foo="
+                Output(SpanKind.Markup);
```

Comment:
Thanks for the [answer](https://github.com/aspnet/Razor/pull/394#discussion_r30569997) below.